stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -555,14 +555,29 @@ function versioninfo(io::IO=stdout)
         "JULIA_NUM_THREADS",
         "MKL_DYNAMIC",
         "MKL_NUM_THREADS",
-        "OPENBLAS_NUM_THREADS",
+         # OpenBLAS has a hierarchy of environment variables for setting the
+         # number of threads, see
+         # https://github.com/xianyi/OpenBLAS/blob/c43ec53bdd00d9423fc609d7b7ecb35e7bf41b85/README.md#setting-the-number-of-threads-using-environment-variables
+        ("OPENBLAS_NUM_THREADS", "GOTO_NUM_THREADS", "OMP_NUM_THREADS"),
```

Comment:
I still think it is not a good idea to look for `OMP_NUM_THREADS`. That should only apply if OpenBLAS is compiled with OpenMP.

Reply:
No, OpenBLAS _always_ looks at that variable, and _only_ that one if it's compiled with `USE_OPENMP=1`: https://github.com/xianyi/OpenBLAS/blob/c43ec53bdd00d9423fc609d7b7ecb35e7bf41b85/README.md#setting-the-number-of-threads-using-environment-variables

Reply:
Right, I saw that comment. Let's do it (as much as it is ugly) :-)

Reply:
Yeah, I'm not saying it's great, it's just how it is 🙂  My guess is that in HPC it's very common to set `OMP_NUM_THREADS` and hope that'd fix the number of threads of all applications (even if that's not necessarily correct) and OpenBLAS accomodates that tradition.

Reply:
BTW, I have wondered often if OMP is good enough for us to use it for our OpenBLAS build, since that would make it more composable with all the other Yggdrasil things we compile with OMP.